src/graphorm/index.ts

```diff
@@ -272,6 +273,17 @@ const obj = new GraphORM({
         select: 'tagsStr',
         transform: (value: string): string[] => value?.split(',') ?? [],
       },
+      clickbaitTitleDetected: {
+        rawSelect: true,
+        select: `"contentQuality"->'is_clickbait_probability'`,
+        transform: (value: string): boolean => {
+          const probability = parseFloat(value || '0');
```

Comment:
Is JSON value not jsonb float/number value already?

Reply:
I guess just to be safe?

Reply:
Just to be safe